zeppelin-web/src/app/interpreter/interpreter.html

```diff
@@ -62,15 +62,7 @@ <h3 class="new_h3">
       <p>Available repository lists. These repositories are used to resolve external dependencies of interpreter.</p>
       <ul class="noDot">
         <li class="liVertical" ng-repeat="repo in repositories">
-          <a tabindex="0" class="btn btn-info" role="button"
-             popover-trigger="focus"
```

Comment:
Checked tooltip now works well. +1
But the tooltip for the first repo button is trimmed. Can we set tooltip position for repo buttons?

![image](https://user-images.githubusercontent.com/4968473/27505650-24afdef8-58e0-11e7-9b39-81559825e669.png)